docs/releasing/publishing.md

```diff
@@ -19,9 +19,9 @@ Developers should pair on releases. When remote working, it can be useful to be
    - changing the 'Unreleased' heading to the new version number and release type. For example, '3.11.0 (Feature release)'
    - adding a new 'Unreleased' heading above the new version number and release type, so users will know where to add PRs to the changelog
 
-6. Update [`package/package.json`](../../package/package.json) version with the new version number.
+6. Run `npm version [version-number] --no-git-tag-version --workspace package` with the new version number.
 
-7. Save the changes. Do not commit.
+7. Do not commit the changes to `package.json` and `package-lock.json`.
```

Comment:
Does this mean we _must_ commit the CHANGELOG changes?

Reply:
Ahh I thought I was being helpfully specific here

Updates to those files are automatic so no "Save the changes" is needed, how about:

```suggestion
7. Do not commit the changes.
```

Reply:
Ah right, so I can update the CHANGELOG, leave it unsaved, and the automatic bit will save and commit that WIP file? That's cool!

Reply:
Ahh sorry no, it's because you had to remember to save the new version in the package previously

Still have save the CHANGELOG

Reply:
I'd probably just keep the existing "Save the changes. Do not commit". I think the mistake it's trying to prevent is somebody forgetting to save a file, so just an extra guard, really.

Reply:
No, wait, bit of a brainfart. Let's just add in an explicit "Save your changes" addendum to step 5?

Reply:
Good idea 😊

I've added a bullet to the CHANGELOG step:

>- saving your changes

But also expanded the `npm version` step so it's nice and clear:

>6. Apply the new version number by running:
>
>   ```
>   npm version <NEW VERSION NUMBER> --no-git-tag-version --workspace package
>   ```
>
>   This step will update the `package.json` and project `package-lock.json` files.
>   
>   Do not commit the changes.